src/widget/sidebar_filler.h

```diff
@@ -0,0 +1,13 @@
+#ifndef WIDGET_SIDEBAR_FILLER_H
+#define WIDGET_SIDEBAR_FILLER_H
+
+#include "input/mouse.h"
+
+void sidebar_filler_draw_filler_foreground(int x_offset, int y_offset, int sidebar_width, int is_collapsed);
+void sidebar_filler_draw_filler_background(int x_offset, int y_offset, int sidebar_width, int is_collapsed);
+
+int sidebar_filler_get_filler_height(void);
```

Comment:
Naming: the function name contains "filler" twice now. Just `sidebar_filler_draw_foreground` will do and is in line with the rest of the project's naming. Same goes for the other two functions.